docs/datamodel/modules.rst

```diff
@@ -15,12 +15,11 @@ Schema objects can be referred to by a fully-qualified name using the
 
 Every EdgeDB schema contains the following standard modules:
 
-- ``std``: all standard types, functions and other declarations;
-- ``schema``: types describing the introspection schema;
-- ``__graphql__``: GraphQL-related types
-
-
-DDL
-===
-
-Module can be defined using the :eql:stmt:`CREATE MODULE` EdgeQL command.
+- ``std``: all standard types, functions and other declarations
+- ``schema``: types describing the introspection schema
+- ``sys``: system-wide entities, such as user roles and
+  :ref:`databases <ref_datamodel_databases>`
+- ``cfg``: configuration and port settings
```

Comment:
"configuration and port settings" -> "configuration and settings"

Reply:
OK